src/payment/is-vaulted-instrument.ts

```diff
@@ -1,5 +1,11 @@
-import { PaymentInstrument, VaultedInstrument } from './payment';
+import { AdyenV2Instrument, FormattedPayload, PaymentInstrument, VaultedInstrument } from './payment';
 
 export default function isVaultedInstrument(instrument: PaymentInstrument): instrument is VaultedInstrument {
+    const formattedPayload = (instrument as FormattedPayload<AdyenV2Instrument>).formattedPayload;
+
+    if (formattedPayload && formattedPayload.bigpay_token) {
+        return Boolean(formattedPayload.bigpay_token.token);
```

Comment:
Ok, This change is required because in `payment-request-transformer.ts` is in use `isVaultedInstrument` and in this step the paymentData is all formattedPayload variable, so is not  possible to cast as VaultedInstrument, that's the reason why I added this validation for Adyen that use formattedPayload